get_staples.hh

```diff
@@ -9,9 +9,10 @@
 
 template<class T, class S> void get_staples(T &K, gaugeconfig<S> &U,
                                    vector<size_t> const x,
-                                   const size_t mu) {
+                                   const size_t mu, const double xi=1.0) {
   vector<size_t> x1 = x, x2 = x;
   x1[mu] += 1;
+  if(xi==1.0){
```

Comment:
okay, that's kind of dangerous to compare floating point numbers with `==`. Let us think whether or not it might be useful to have a boolean switch parameter in addition to `xi`. This would also allow one to compare the two code part for `xi=1`, where they should give identical results (up to rounding).